server/container/guice/pom.xml

```diff
@@ -107,6 +107,11 @@
             </dependency>
             <dependency>
                 <groupId>${project.groupId}</groupId>
+                <artifactId>james-server-guice-webadmin-swagger</artifactId>
```

Comment:
No.

Bind this in guice implementation (as we don't want to enforce the use of webadmin dependency everywhere)

You should do that in cassandra-guice, guice-jpa, jpa-smtp, memory, etc... project instead.